backEnd/repositories/userRepository.js

```diff
@@ -9,12 +9,12 @@ admin.initializeApp({
 const db = admin.firestore();
 
 module.exports = {
-  
-  createUser: async function(toCreate) {
-    console.log('createUser: ' + toCreate);
-    let created = await db.collection('users').doc(toCreate.id).set(toCreate);
-    console.log('created user: ' + created);
-  },
+  // has been ported over to functions
+  // createUser: async function(toCreate) {
```

Comment:
probably good to leave this for now until we confirm that backend isn't needed anymore